artview/components/nav.py

```diff
@@ -258,40 +266,91 @@ def AdvanceFileSelect(self, findex):
             return
         self.fileindex = findex
         self.filename = self.Vfilelist.value[findex]
-        self.menu._openfile(filename=self.filename)
-        self._update_InfoUI()
```

Comment:
need not to update info. Info is update when Vradar or Vgrid is changed. So opening a new file will automatically do it